src/components/Confirm/index.js

```diff
@@ -57,35 +58,48 @@ class CaptureViewer extends Component {
   }
 }
 
-
-const Previews = ({capture, retakeAction, confirmAction} ) =>
-  <div className={`${theme.previews} ${theme.step}`}>
+const PreviewHeader = () =>
+  <div>
     <h1 className={theme.title}>Confirm capture</h1>
     <p>Please confirm that you are happy with this photo.</p>
-    <CaptureViewer capture={capture} />
-    <div className={`${theme.actions} ${style.actions}`}>
-      <button
-        onClick={retakeAction}
-        className={`${theme.btn} ${style["btn-outline"]}`}
-      >
-        Take again
-      </button>
-      <a
-        href=''
-        className={`${theme.btn} ${theme["btn-primary"]}`}
-        onClick={preventDefaultOnClick(confirmAction)}
-      >
-        Confirm
-      </a>
+  </div>
+
+const RetakeAction = ({retakeAction}) =>
+  <button onClick={retakeAction}
+    className={`${theme.btn} ${style["btn-outline"]}`}>
+    Take again
+  </button>
+
+const ConfirmAction = ({confirmAction, error}) => {
+  if (!error) return (
+    <a href='' className={`${theme.btn} ${theme["btn-primary"]}`}
```

Comment:
`href="#"` is generally the standard for making a link that is clickable but doesn't _go_ anywhere

You _can_ just style the link correctly to still be underlined etc., but I think you're making things harder for screen-readers by having a clickable empty-href anchor

See [this StackOverflow](https://stackoverflow.com/questions/5292343/is-an-anchor-tag-without-the-href-attribute-safe) post about placeholder hyperlinks